instrumentation/jdbc/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jdbc/DbRequest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.jdbc;
+
+import static io.opentelemetry.javaagent.instrumentation.jdbc.JdbcUtils.connectionFromStatement;
+import static io.opentelemetry.javaagent.instrumentation.jdbc.JdbcUtils.extractDbInfo;
+
+import com.google.auto.value.AutoValue;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.Statement;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+@AutoValue
+public abstract class DbRequest {
+  public abstract DbInfo getDbInfo();
```

Comment:
Do we use DbInfo to reduce the scope of this PR? I guess it could be inlined in

Reply:
I thought about inlining it first, but I noticed that it's being cached per `Connection`: see `JdbcMaps.connectionInfo`. I thought it would be simpler to just create another small class for request and reuse `DbInfo` instances.